server/constants/eventAudit.ts

```diff
@@ -7,6 +7,7 @@ const eventAuditTypes: Partial<Record<EventAuditType, string>> = {
   REQUESTED_VISIT: 'Requested',
   REQUESTED_VISIT_APPROVED: 'Approved',
   REQUESTED_VISIT_REJECTED: 'Rejected',
+  REQUESTED_VISIT_AUTO_REJECTED: 'Rejected',
```

Comment:
This is the label on the visit history timeline. Assuming this should be `Rejected` and not something like `Auto-rejected`? Can't see any relevant design...